docker/jagger/webclient/Dockerfile

```diff
@@ -1,17 +1,23 @@
-FROM openjdk:8u102-jdk
+FROM openjdk:8u92-jdk-alpine
```

Comment:
The main difference for us is the size of distributive. 8u92-jdk-alpine three times less than 8u102-jdk but contains all that is necessary for us.